code/modules/projectiles/guns/projectile/automatic/nordwind.dm

```diff
@@ -46,8 +46,8 @@
 
 /obj/item/gun/projectile/automatic/nordwind/strelki
 	name = "\"Strelki\" precision rifle"
-	desc = "A Strelki precision rifle, an inexpensive rifle of desent quality made by Hunt Inc.\
-			Its mostly wooden design and its usage of 7.62mm rounds make it the ideal weapon for a hunter tired of a crude boltaction. The frontier's favorite sniper!"
+	desc = "A Strelki precision rifle, an inexpensive rifle of desent quality made by Hunters Inc.\
```

Comment:
```suggestion
	desc = "A Strelki precision rifle, an inexpensive rifle of desent quality made by Hunt Inc.\
```

Reply:
If you check at this weapon's serial numbers, it reads "Hunters Inc" instead of "Hunt Inc", that's why I'm using the proper licensing name here, also thanks I never noticed it says "desent" instead of "decent" until now

Reply:
![image](https://user-images.githubusercontent.com/30435998/217689806-9a09f599-43a8-4c14-afcd-a5f4a8b216c2.png)